appinfo/routes.php

```diff
@@ -58,6 +60,11 @@
 // general API
 ['name' => 'api#index', 'url' => '/api', 'verb' => 'GET'],
 
+// API 2
+['name' => 'folder_api_v2#create_folder', 'url' => '/api/v2/folders', 'verb' => 'POST'],
+['name' => 'folder_api_v2#update_folder', 'url' => '/api/v2/folders/{folderId}', 'verb' => 'PATCH'],
+['name' => 'folder_api_v2#delete_folder', 'url' => '/api/v2/folders/{folderId}', 'verb' => 'DELETE'],
```

Comment:
Maybe we should use the function names from: https://docs.nextcloud.com/server/20/developer_manual/basics/routing.html#registering-resources

Reply:
I'm a fan of clear function names which tell me exactly what they do, even if it means to write more chars and repeat something. But this is of course a matter of taste and the "rules" of a repository.. :wink:

Reply:
I don't object to clarity of course. But I'd say `FolderController::create` is pretty clear on what it's going to create 😄